retroshare-gui/src/gui/TheWire/PulseMessage.cpp

```diff
@@ -148,7 +148,7 @@ void PulseMessage::setRefImageCount(uint32_t count)
 	}
 }
 
-void PulseMessage::viewPicture()
+void PulseMessage::viewPictureOne()
```

Comment:
Plz use sender() to get which widgets sends the signal. This way you can choose which picture to show and avoid duplicating all the code!